etna/commands/backtest_command.py

```diff
@@ -19,6 +23,12 @@ def backtest(
     freq: str = typer.Argument(..., help="frequency of timestamp in files in pandas format"),
     output_path: Path = typer.Argument(..., help="where to save forecast"),
     exog_path: Optional[Path] = typer.Argument(default=None, help="path to csv with exog data"),
+    known_future: Optional[List[str]] = typer.Argument(
+        None,
```

Comment:
why not `default="all"`?

Reply:
Because "all" is not List. And Typer does not support Union. So I can only configure it like this `"all" if not known_future else known_future`

Reply:
Why not?